tests/Behat/Context/UI/CustomerOrderCommentsContext.php

```diff
@@ -51,6 +51,15 @@ public function iCommentTheOrderWith(OrderInterface $order, string $message): vo
         $this->orderCommentFormElement->comment();
     }
 
+    /**
+     * @Then the notify customer checkbox is enabled
+     * @When the notify customer checkbox is enabled
```

Comment:
Scenarios places in *.feature files imply that you use this step only in `Given` section.

Reply:
Can you elaborate? I don't quite get that. :(

Reply:
What I mean is that you can replace 

```
* @Then the notify customer checkbox is enabled
* @When the notify customer checkbox is enabled
```

with 

```
* @Given the notify customer checkbox is enabled
```